src/streaming/controllers/EventController.js

```diff
@@ -80,6 +80,11 @@ function EventController() {
     }
 
     function stop() {
+        // dispatch any lingering events
+        if (Object.keys(inlineEvents).length > 0 || Object.keys(inbandEvents).length > 0) {
+            onEventTimer();
+        }
```

Comment:
Occasionally stop is called before the all of the events have been fired. Manually call the update method if there are lingering events.